children/schema.py

```diff
@@ -264,7 +300,7 @@ def validate_child_data(child_data):
     if "birthdate" in child_data:
         birth_year = child_data["birthdate"].year
         if (
-            child_data["birthdate"] > localtime(now()).date()
+            child_data["birthdate"] > localtime(timezone.now()).date()
```

Comment:
django.utils seems to have `localdate` (See [docs](https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.timezone.localdate)) which should give today intepreted in local time if nothing is passed to it.

Reply:
True, changing to that 👍